core/src/main/java/io/apicurio/hub/api/codegen/pre/OpenApiAdditionalPropertiesDataTypeProcessor.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.hub.api.codegen.pre;
+
+import io.apicurio.datamodels.combined.visitors.CombinedVisitorAdapter;
+import io.apicurio.datamodels.core.models.common.IDefinition;
+import io.apicurio.datamodels.openapi.models.OasSchema;
+
+/**
+ * @author eric.wittmann@gmail.com
+ */
+public class OpenApiAdditionalPropertiesDataTypeProcessor extends CombinedVisitorAdapter {
```

Comment:
I'm not sure this class is needed

Reply:
I will check when a new release of jsonschema2pojo is out with your fix, and thanks for that!

Reply:
That seems reasonable to me. Regarding this particular class, won't it overwrite the additional properties schema with a boolean? I'm thinking the additional properties in the schema should left as-is for the POJO generator.

Reply:
Not overriding the boolean, unfortunately, triggers the bug you already fixed upstream.

Reply:
...very true ...

Reply:
Sorry for the delay, but I fully applied the workaround for now to unblock this PR.